File: articles/anchoring-bias-economic-judgment/go/anchoring_adjustment.go
package main

import "fmt"

func anchoredEstimate(anchor, trueValue, adjustmentRate float64) float64 {
	return anchor + adjustmentRate*(trueValue-anchor)
}

func anchoringBias(anchor, trueValue, adjustmentRate float64) float64 {
	return anchoredEstimate(anchor, trueValue, adjustmentRate) - trueValue
}

func main() {
	fmt.Printf("Anchored estimate: %.2f\n", anchoredEstimate(85.0, 65.0, 0.55))
	fmt.Printf("Anchoring bias: %.2f\n", anchoringBias(85.0, 65.0, 0.55))
}

File: articles/anchoring-bias-economic-judgment/fortran/anchoring_adjustment.f90
program anchoring_adjustment
  implicit none
  real :: anchor, true_value, adjustment_rate, estimate, bias

  anchor = 85.0
  true_value = 65.0
  adjustment_rate = 0.55

  estimate = anchor + adjustment_rate * (true_value - anchor)
  bias = estimate - true_value

  print *, "Anchored estimate:", estimate
  print *, "Anchoring bias:", bias
end program anchoring_adjustment
